libs/ledger-live-common/src/hw/connectApp.ts

```diff
@@ -429,15 +429,6 @@ const cmd = ({ deviceId, request }: Input): Observable<ConnectAppEvent> => {
               }
             }),
             catchError((e: unknown) => {
-              if (
-                e instanceof DisconnectedDeviceDuringOperation ||
-                e instanceof DisconnectedDevice
```

Comment:
You can remove the import of DisconnectedDeviceDuringOperation and DisconnectedDevice ;) 

Though by doing this it will have an impact on some other device actions (not just installLanguage) that might rely on this `disconnected` event. Are we sure we need to remove this check ?

Reply:
It was done in order to fix a QA review on a different case that LIVE-8896 specified. Wouldn't it better to create another task for those commits @aussedatlo ?

Reply:
Indeed, it could be dangerous.

Let me remove this part, like @jdabbech-ledger suggest, it's better to fix this case in another ticket